src/classes/PMT_PaymentWizard_CTRL.cls

```diff
@@ -320,7 +320,7 @@ public with sharing class PMT_PaymentWizard_CTRL {
         for (String s : PMT_PaymentCreator.paymentMappings.keyset()) {
             npe01__Payment_Field_Mapping_Settings__c pfms = PMT_PaymentCreator.paymentMappings.get(s);
 
-            if (!query.contains(pfms.npe01__Opportunity_Field__c.toLowerCase())) {
+            if (!query.toLowerCase().contains(pfms.npe01__Opportunity_Field__c.toLowerCase())) {
```

Comment:
The change in the payment wizard file does not seem to apply to the issue being fixed. Can you please revert it back if it is not required?